rust/axum/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "server"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+axum = "0.4.3"
```

seems too strict for me, I think 
```suggestion
axum = "~0.4"
```

should be better, to make sure we are up-to-date but using 0.4.x (their is a bot taking care of updates here)

---

Cargo treats `0.4.3` as `^0.4.3`, not `=0.4.3`. `~0.4` would additionally allow *older* versions than 0.4.3, newer ones would be chosen with either version bound (when there is no lockfile or when `cargo update` is run).

---

I assume then this will keep into 0.4.x, that the he only thing to notice here.

I think better to stick to `rust` best practices, I let then you decide guys (I'm more a `ruby` dev and we do not have `^` constraints)

---

Yeah, `^0.x.y` for some x > 0 means `>=0.x.y && <0.(x+1).0` in Rust & NodeJS land.